fw/http_parser.c

```diff
@@ -9807,6 +9845,11 @@ tfw_h2_parse_req_hdr(unsigned char *data, unsigned long len, TfwHttpReq *req,
 			     __h2_req_parse_x_forwarded_for,
 			     TFW_HTTP_HDR_X_FORWARDED_FOR, 0);
 
+	/* 'te' is read, process field-value */
+	TFW_H2_PARSE_HDR_VAL(Req_HdrTeV, msg,
+			     __h2_req_parse_te,
+			     TFW_HTTP_HDR_RAW, 1);
```

Comment:
Please add unit tests for the new parsing logic to `fw/t/unit/test_http2_parser.c`